cmake/CMakeLists.txt

```
@@ -0,0 +1,160 @@
+project(usb-1.0)
```

no need to initialize `CXX` environment
```suggestion
project(usb-1.0 C)
```